app/main.py

```diff
@@ -1 +1,45 @@
-#  write your code here
+from __future__ import annotations
+
+from typing import List
+
+
+class Animal:
+    def __init__(
+            self, name: str, appetite: int, is_hungry: bool = True
```

Comment:
Place each argument on a new line to improve the readability in the `__init__` through all classes.